### PR TITLE
Reverse Proxy Single Host Example

### PR DESCRIPTION
**Pull request by girisairam450**

### CODE REVIEW

File: examples/reverse-proxy/mnist/nginx.conf [hunk 1]
```diff
@@ -0,0 +1,74 @@
+######################################################################
+## (C)Copyright 2022 Hewlett Packard Enterprise Development LP
+######################################################################
+
+# user  nginx;
+
+# Default value of worker_processes will be 1 if not specified. Auto will adjust he number of worker
```
Comment (suresh-ls) [inline examples/reverse-proxy/mnist/nginx.conf:7]: th -> the

**Review by suresh-ls — APPROVED** [review]

**Review by htjain — APPROVED** [review]

File: examples/reverse-proxy/README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 1.  Reverse proxy examples use NGINX and BIND9 as custom Docker images to build it. Both of these images use Ubuntu 22.04 as the base image, along with some image-specific apt-get packages. Make sure these packages and images are downloaded properly while running the scripts within these examples.
 
-2.  Before running these examples, make sure APLS is already running.
+2.  Before running these examples, make sure APLS is already running. Also, ensure wheel file with its symbolic link are placed under the `lib` directory.
```
Comment (iArpanPatel) [inline examples/reverse-proxy/README.md:8]: Can we add this into main Example README under 'Prerequisite for all examples'?

Comment (girisairam450) [reply examples/reverse-proxy/README.md:8]: Done

**Comment by IMOKURI** [comment]
Hello,
Thank you for adding the reverse proxy example.

In this example, the nginx and SN nodes are on the same host.
I think that nginx and SN nodes are often on different hosts.

In that case, can the `--sn-p2p-service` option of the `run-sn` command specify the host where nginx is running?
Or do I need to specify the host where the SN node is running?

**Comment by girisairam450** [comment]
Hi, [SUGIYAMA Yoshio](https://github.com/IMOKURI), Thanks for trying out the reverse proxy example. While we check and get back to you about the Nginx and SN on different hosts query, Can you please initiate this to Discussions->Q&A. So that it will be helpful for all of the community?

**Comment by IMOKURI** [comment]
Thank you for your reply. I've created discussion.

https://github.com/HewlettPackard/swarm-learning/discussions/150

**Pull request merged**